gax/src/main/java/com/google/api/gax/rpc/FixedWatchdogProvider.java

```diff
@@ -31,13 +31,15 @@
 
 import com.google.api.core.ApiClock;
 import com.google.api.core.BetaApi;
+import com.google.api.core.InternalApi;
 import java.util.concurrent.ScheduledExecutorService;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import org.threeten.bp.Duration;
 
 @BetaApi("The surface for streaming is not stable yet and may change in the future.")
-public class FixedWatchdogProvider implements WatchdogProvider {
+@InternalApi
```

Comment:
What side effects will external users see as a result of changing this from BetaApi to InternalApi?

Reply:
Strictly speaking it does not stop being beta API (the beta annotation is still there), it becomes both beta and internal. It is a conventional thing, so nothing should break because of that. 

As for conventions, with both of these annotations present, people should be even more discouraged to use these classes in their code. I.e. `@BetaApi` may mean that this api will change in the future, and adding `@InternalApi` annotation indicates that the change has happened, and now the users are definitely not supposed to use this class in their code.